packages/sdk/tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "declarationMap": true,
     "skipLibCheck": true,
     "target": "es2015",
-    "noImplicitAny": true,
+    "noImplicitAny": false,
```

Comment:
Any reason why this changed?

Reply:
ah, i was fooling around with some tests, and meant to change this back to `true`. good catch